index.html

```diff
@@ -11,39 +11,212 @@
 
 <body>
     <div class="container-parent">
-
         <header class="top-navigation">
             <div class="container-logo">
                 <a href="#" class="logo">My Logo</a>
             </div>
-            <nav>
-                <img class="menu-icon" src="./assets/mobile/Icon - Menu.svg" alt="Menu Button">
+            <nav class="menu-icon">
+                <img src="./assets/mobile/Icon - Menu.svg" alt="Menu Button">
             </nav>
         </header>
-
         <main class="main">
             <section class="headline">
                 <div class="text-primary">
-                    <h1 class="primary-one">I’m Porag. Glad to see you!</h1>
+                    <h1 class="primary-one">I’m Porag. <br>Glad to see <br>you!</h1>
                     <p class="primary-two">I’m a software developer! I can help you build a product , feature or website
                         Look through some of my work and experience! If you like what you see and have a project you
                         need coded, don’t hesitate to contact me.</p>
                 </div>
```

Comment:
- [ ] As required for this project The section uses the font and colors (background and font) should defined for the template, but I can see that your paragraph doesn't reflect the figma design kindly fix this :wink:  :point_down:  the font are not looking the same 
Your Design 
![Screenshot from 2023-03-26 00-52-54](https://user-images.githubusercontent.com/88894525/227746216-312166f7-aba4-4fd4-8fdb-9eae9cb77d47.png)

Figma Design
![Screenshot from 2023-03-26 00-56-46](https://user-images.githubusercontent.com/88894525/227746274-93c14f3d-2930-4e18-a293-d1a9994f7e99.png)

Reply:
As this was part of the first pull request which was the "Hello Microverse" project, so can this required change be an approval blocker for this particular part? 
It would be great if you could shed some light on this, please. Thanks

Reply:
Hello @porag-m06 Your design should keep matching the figma design as required for this project 😊 Good question and you can't get approved until your project get completed